addons/jobs/jobs-service/src/main/java/org/kie/kogito/jobs/service/model/ScheduledJob.java

```diff
@@ -16,20 +16,31 @@
 
 package org.kie.kogito.jobs.service.model;
 
+import java.time.ZonedDateTime;
 import java.util.Optional;
-import java.util.StringJoiner;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
 import org.kie.kogito.jobs.api.Job;
+import org.kie.kogito.jobs.service.utils.DateUtil;
 
+@RegisterForReflection
```

Comment:
Don’t think this is needed, we won’t run it as naitive executable

Reply:
Yeah, my bad I forgot to remove it.
Done